spec/index.html

```diff
@@ -1498,6 +1498,51 @@ <h2>RDF Datasets</h2>
     between datasets would require RDF graph entailments between the graphs with the same name
     (adding in empty graphs as required).</p>
 
+  <table>
+    <caption>Definition of an RDF dataset</caption>
+    <tr>
+       <td class="semantictable">
+       <p>An <dfn data-lt="RDF dataset">RDF dataset</dfn> I is a set:</p>
+              { G, (&lt;u<sub>1</sub>&gt;, G<sub>1</sub>), (&lt;u<sub>2</sub>&gt;, G<sub>2</sub>), . .
+              . (&lt;u<sub>n</sub>&gt;, G<sub>n</sub>) }<br>
+              where n&ge;0 and G and each G<sub>i</sub> are graphs, and each &lt;u<sub>i</sub>&gt; is an IRI. Each
+              &lt;u<sub>i</sub>&gt; is distinct.</p>
+            <p>G is called the default graph. (&lt;u<sub>i</sub>&gt;, G<sub>i</sub>) are called named
+              graphs.</p>
+        </td>
+    </tr>
+  </table>
+
+  <table>
+    <caption>Definition of RDF dataset merge</caption>
+    <tr>
+       <td class="semantictable">
+            <p>Let DS1 = { G1, (&lt;u1<sub>1</sub>&gt;, G1<sub>1</sub>), (&lt;u1<sub>2</sub>&gt;,
+              G1<sub>2</sub>), . . . (&lt;u1<sub>n</sub>&gt;, G1<sub>n</sub>) },<br>
+              and DS2 = { G2, (&lt;u2<sub>1</sub>&gt;, G2<sub>1</sub>), (&lt;u2<sub>2</sub>&gt;,
+              G2<sub>2</sub>), . . . (&lt;u2<sub>m</sub>&gt;, G2<sub>m</sub>) }</p>
+            <p>The <dfn data-lt="RDF dataset">RDF dataset merge</dfn> of DS1 and DS2 is:<br>
```

Comment:
This actually becomes a duplicate definition of RDF dataset, whereas I believe you want a definition of "RDF Dataset merge".

```suggestion
            <p>The <dfn>RDF dataset merge</dfn> of DS1 and DS2 is:<br>
```

Reply:
Should be fixed in latest commit.